lib/mzgl/gl/Graphics.h

```diff
@@ -73,6 +76,9 @@ class Graphics {
 	enum class BlendMode {
 		Alpha, // the classic alpha blending mode
 		Additive,
+		Multiply,
+		Screen,
+		Subtract,
```

Comment:
we're mostly using UpperCamelCase for enum classes - see Vbo::PrimitiveMode and the SP404 stuff - maybe it would be nicer for them to be lowerCamelCase - but it would be good to do that on a pass of the develop branch